components/automate-ui/src/app/pages/+compliance/+reporting/reporting-searchbar/reporting-searchbar.component.html

```diff
@@ -88,7 +88,7 @@
     primary
     *ngFor="let filter of filters"
     (click)="onRemoveFilterClick(filter)">
-    <span>{{ filter.type.name }}: {{ filter.value.text }}</span>
+    <span>{{ filter.type.name }}: {{ displayText(filter.value.text) }}</span>
```

Comment:
The API call for these filters is built using the `filter.value.text`, so if I changed that text earlier on for use by the badge, it would affect the API call.   I instead decided to to use a small function that checks if the string is empty or not, and if it is empty, it returns 'no value'.  This function is only used for displaying the "Filtered By" badges.